README.md

```diff
@@ -14,27 +14,27 @@ This project leverages [flutter_rust_bridge](https://cjycode.com/flutter_rust_br
 
 ## Getting Started
 
-1. Add the velopack_flutter dependency to your `pubspec.yaml`:
+1. Make sure rust is installed: https://www.rust-lang.org/tools/install
+
+2. Add the velopack_flutter dependency to your `pubspec.yaml`:
 
 ```yaml
 dependencies:
   velopack_flutter: ^0.0.1
 ```
 
-2. Import the package, initialize the Rust library and handle Velopack app hooks in your `main.dart`:
+3. Import the package and initialize the Rust library `main.dart`:
 
 ```dart
 import 'package:flutter/material.dart';
 import 'package:velopack_flutter/velopack_flutter.dart';
 
 Future<void> main() async {
   await RustLib.init();
-  
-  final veloCommands = ['--veloapp-install', '--veloapp-updated', '--veloapp-obsolete', '--veloapp-uninstall'];
```

Comment:
Are you sure these can be removed? As far as I can tell these are required to ensure updates and installs are properly handled, see https://docs.velopack.io/integrating/hooks

Reply:
In the example they were not hooked up to anything (  if (veloCommands.any((cmd) => args.contains(cmd))) {  <<<<- its not shown where "args" comes from). I am not certain about anything here.  How do you listen to these events?

Reply:
The args actually come from starting your flutter project like this "your_exe.exe --veloapp-install", the velopack updater does this automatically as part of the update process and expects them to be handled as mentioned in the link above

Reply:
but you cannot compile the application without specifying args somewhere. Are args supposed to be in the main function signature then?

Reply:
Ah I see, yeah it should be `Future<void> main(List<String> args) async, didn't notice it before